### ISSUE TITLE
Raised JSON Exception: options :symbolize_names and :create_additions cannot be  used in conjunction (ArgumentError)

### ISSUE THREAD

**Issue by ionutciordas** [yujinakayama/transpec#123]
Hello!

Environment:
```
➜ gem environment
RubyGems Environment:
  - RUBYGEMS VERSION: 2.0.3
  - RUBY VERSION: 2.0.0 (2013-06-27 patchlevel 247) [x86_64-darwin14.0.0]
  - INSTALLATION DIRECTORY: /Users/ionutciordas/.rbenv/versions/2.0.0-p247/lib/ruby/gems/2.0.0
  - RUBY EXECUTABLE: /Users/ionutciordas/.rbenv/versions/2.0.0-p247/bin/ruby
  - EXECUTABLE DIRECTORY: /Users/ionutciordas/.rbenv/versions/2.0.0-p247/bin
  - RUBYGEMS PLATFORMS:
    - ruby
    - x86_64-darwin-14
  - GEM PATHS:
     - /Users/ionutciordas/.rbenv/versions/2.0.0-p247/lib/ruby/gems/2.0.0
     - /Users/ionutciordas/.gem/ruby/2.0.0
  - GEM CONFIGURATION:
     - :update_sources => true
     - :verbose => true
     - :backtrace => false
     - :bulk_threshold => 1000
  - REMOTE SOURCES:
     - https://rubygems.org/
➜ which transpec
/Users/ionutciordas/.rbenv/shims/transpec
➜ transpec --version
3.2.2
➜ bundle exec rspec --version
2.99.2
➜ ruby --version
ruby 2.0.0p247 (2013-06-27 revision 41674) [x86_64-darwin14.0.0]
➜ rails -v
Rails 3.2.22.2
```

I get the following error when running:
```
➜ transpec
Copying the project for dynamic analysis...
Running dynamic analysis with command "bundle exec rspec"...
.................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................

Finished in 15.47 seconds
753 examples, 0 failures
/Users/ionutciordas/.rbenv/versions/2.0.0-p247/lib/ruby/gems/2.0.0/gems/json-2.0.2/lib/json/common.rb:156:in `initialize': options :symbolize_names and :create_additions cannot be  used in conjunction (ArgumentError)
	from /Users/ionutciordas/.rbenv/versions/2.0.0-p247/lib/ruby/gems/2.0.0/gems/json-2.0.2/lib/json/common.rb:156:in `new'
	from /Users/ionutciordas/.rbenv/versions/2.0.0-p247/lib/ruby/gems/2.0.0/gems/json-2.0.2/lib/json/common.rb:156:in `parse'
	from /Users/ionutciordas/.rbenv/versions/2.0.0-p247/lib/ruby/gems/2.0.0/gems/json-2.0.2/lib/json/common.rb:335:in `load'
	from /Users/ionutciordas/.rbenv/versions/2.0.0-p247/lib/ruby/gems/2.0.0/gems/transpec-3.2.2/lib/transpec/dynamic_analyzer/runtime_data.rb:17:in `load'
	from /Users/ionutciordas/.rbenv/versions/2.0.0-p247/lib/ruby/gems/2.0.0/gems/transpec-3.2.2/lib/transpec/dynamic_analyzer.rb:110:in `block in load_analysis_result'
	from /Users/ionutciordas/.rbenv/versions/2.0.0-p247/lib/ruby/gems/2.0.0/gems/transpec-3.2.2/lib/transpec/dynamic_analyzer.rb:109:in `open'
	from /Users/ionutciordas/.rbenv/versions/2.0.0-p247/lib/ruby/gems/2.0.0/gems/transpec-3.2.2/lib/transpec/dynamic_analyzer.rb:109:in `load_analysis_result'
	from /Users/ionutciordas/.rbenv/versions/2.0.0-p247/lib/ruby/gems/2.0.0/gems/transpec-3.2.2/lib/transpec/dynamic_analyzer.rb:45:in `block in analyze'
	from /Users/ionutciordas/.rbenv/versions/2.0.0-p247/lib/ruby/gems/2.0.0/gems/transpec-3.2.2/lib/transpec/dynamic_analyzer.rb:52:in `in_copied_project'
	from /Users/ionutciordas/.rbenv/versions/2.0.0-p247/lib/ruby/gems/2.0.0/gems/transpec-3.2.2/lib/transpec/dynamic_analyzer.rb:40:in `analyze'
	from /Users/ionutciordas/.rbenv/versions/2.0.0-p247/lib/ruby/gems/2.0.0/gems/transpec-3.2.2/lib/transpec/cli.rb:75:in `block in run_dynamic_analysis'
	from /Users/ionutciordas/.rbenv/versions/2.0.0-p247/lib/ruby/gems/2.0.0/gems/transpec-3.2.2/lib/transpec/dynamic_analyzer.rb:26:in `block in initialize'
	from /Users/ionutciordas/.rbenv/versions/2.0.0-p247/lib/ruby/gems/2.0.0/gems/transpec-3.2.2/lib/transpec/dynamic_analyzer.rb:58:in `chdir'
	from /Users/ionutciordas/.rbenv/versions/2.0.0-p247/lib/ruby/gems/2.0.0/gems/transpec-3.2.2/lib/transpec/dynamic_analyzer.rb:58:in `block in in_copied_project'
	from /Users/ionutciordas/.rbenv/versions/2.0.0-p247/lib/ruby/2.0.0/tmpdir.rb:88:in `mktmpdir'
	from /Users/ionutciordas/.rbenv/versions/2.0.0-p247/lib/ruby/gems/2.0.0/gems/transpec-3.2.2/lib/transpec/dynamic_analyzer.rb:56:in `in_copied_project'
	from /Users/ionutciordas/.rbenv/versions/2.0.0-p247/lib/ruby/gems/2.0.0/gems/transpec-3.2.2/lib/transpec/dynamic_analyzer.rb:25:in `initialize'
	from /Users/ionutciordas/.rbenv/versions/2.0.0-p247/lib/ruby/gems/2.0.0/gems/transpec-3.2.2/lib/transpec/cli.rb:73:in `new'
	from /Users/ionutciordas/.rbenv/versions/2.0.0-p247/lib/ruby/gems/2.0.0/gems/transpec-3.2.2/lib/transpec/cli.rb:73:in `run_dynamic_analysis'
	from /Users/ionutciordas/.rbenv/versions/2.0.0-p247/lib/ruby/gems/2.0.0/gems/transpec-3.2.2/lib/transpec/cli.rb:53:in `process'
	from /Users/ionutciordas/.rbenv/versions/2.0.0-p247/lib/ruby/gems/2.0.0/gems/transpec-3.2.2/lib/transpec/cli.rb:38:in `run'
	from /Users/ionutciordas/.rbenv/versions/2.0.0-p247/lib/ruby/gems/2.0.0/gems/transpec-3.2.2/lib/transpec/cli.rb:19:in `run'
	from /Users/ionutciordas/.rbenv/versions/2.0.0-p247/lib/ruby/gems/2.0.0/gems/transpec-3.2.2/bin/transpec:7:in `<top (required)>'
	from /Users/ionutciordas/.rbenv/versions/2.0.0-p247/bin/transpec:23:in `load'
	from /Users/ionutciordas/.rbenv/versions/2.0.0-p247/bin/transpec:23:in `<main>'
```

Any idea what am I missing?
Thank you

**Comment by hayley** [comment]
I'm seeing this issue too.

```
bundle exec rspec --version
3.4.4
transpec --version
3.2.2
```

**Comment by marcbey** [comment]
Works with json 1.8.3

❯ transpec --version
3.2.2

❯ gem list|grep json
json (1.8.3)

❯ rspec --version
3.5.4

**Comment by yujinakayama** [comment]
Fixed in transpec 3.3.0.
